pkg/v1/service/books/service.go

```diff
@@ -3,15 +3,16 @@ package books
 import "context"
 
 type Service interface {
-    GetBook(ctx context.Context, id string) (Book, error)
+	GetBook(ctx context.Context, id string) (Book, error)
```

Comment:
From my previous comment:

```suggestion
	GetBook(ctx context.Context, id string) (*Book, error)
```